src/api/profiles/profilesProfileIdDelete.ts

```diff
@@ -0,0 +1,7 @@
+import {baseRequestService} from "../base.ts";
+import {ProfileById} from "../../types/domain.ts";
+
+export const profilesProfileIdDelete = (id: string) => baseRequestService<ProfileById>({
```

Comment:
```suggestion
export const profilesProfileIdDelete = (id: string) => baseRequestService({
```

в треугольных возвращаемое значение, делит ничо не отдает наскок я помню